src/main/java/fi/aalto/cs/apluscourses/intellij/actions/SelectAllFiltersAction.java

```diff
@@ -0,0 +1,22 @@
+package fi.aalto.cs.apluscourses.intellij.actions;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import fi.aalto.cs.apluscourses.presentation.filter.Options;
+import org.jetbrains.annotations.NotNull;
+
+public class SelectAllFiltersAction extends AnAction {
```

Comment:
The select/deselect toggle isn't enabled during indexing. An easy way to test this: turn a new project into a course project for O1. As O1Libary and Scala SDK is installed, indexing starts happening. The exercises are listed and all other filters are available, but the toggle is disabled. To allow an action run during indexing, it should implement [`DumbAware`](https://upsource.jetbrains.com/idea-ce/file/idea-ce-4b94ba01122752d7576eb9d69638b6e89d1671b7/platform/core-api/src/com/intellij/openapi/project/DumbAware.java).